code/modules/research/designs/circuit/computer_circuits.dm

```diff
@@ -87,3 +87,8 @@
 	name = "Ship Sensors Console"
 	req_tech = list(TECH_ENGINEERING = 3, TECH_DATA = 2)
 	build_path = /obj/item/circuitboard/ship/sensors
+
+/datum/design/circuit/computer/engines
+	name = "Helm Control Console"
```

Comment:
Misleading name. It's not the helm control console, but the engine control console.

Reply:
Ruined by copy+paste, thank you